### PR TITLE
[cmake] Added CMakeLists for generator.

### PR DESCRIPTION
**Pull request by ygorshenin**
@syershov @therearesomewhocallmetim PTAL

### CODE REVIEW

File: generator/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1,93 @@
...
+  tesselator.hpp
+  towns_dumper.cpp
+  towns_dumper.hpp
+  traffic_generator.cpp
+  traffic_generator.hpp
+  unpack_mwm.cpp
+  unpack_mwm.hpp
+  ways_merger.hpp
+  world_map_generator.hpp
+  )
```
Comment (therearesomewhocallmetim) [inline generator/CMakeLists.txt]: Пробелы перед скобкой не нужны

Comment (ygorshenin) [inline generator/CMakeLists.txt]: Done

**Comment by ygorshenin** [comment]
@therearesomewhocallmetim PTAL

**Comment by therearesomewhocallmetim** [comment]
LGTM

**Pull request merged**